python/ray/tests/test_global_state.py

```diff
@@ -357,6 +357,10 @@ def self_ip_is_set():
     global_state_accessor.disconnect()
 
 
+def test_next_job_id(ray_start_regular):
+    assert ray.state.next_job_id().int() + 1 == ray.state.next_job_id().int()
```

Comment:
This looks tricky to me. How about

```
job_id_1 = ray.state.next_job_id()
job_id_2 = ray.state.next_job_id()
assert job_id_1.int() + 1 == job_id_2.int()
```

Reply:
No problem. I wrote like this in first version. :-)